### PR TITLE
added text utilities, lists, navigation and modal components

### PR DESCRIPTION
**Pull request by Hamza-777**
Add feedback and review this PR for me please....

### CODE REVIEW

File: components/card.css [hunk 1]
```diff
@@ -166,8 +166,15 @@ img {
 
 .overlay-text {
     position: absolute;
-    bottom: 3rem;
-    left: 1rem;
+    inset: 0;
+    opacity: 0;
+    display: flex;
+    align-items: center;
+    background: rgba(0, 0, 0, .1);
```
Comment (vrinda-mahajan) [inline components/card.css:173]: Use variables for colors.

File: components/navigation.css [hunk 2]
```diff
@@ -0,0 +1,52 @@
+.nav-container {
+    position: relative;
+    max-width: 900px;
+    height: 300px;
```
Comment (vrinda-mahajan) [inline components/navigation.css]: Use rem in place of px for responsiveness.

**Review by vrinda-mahajan — CHANGES_REQUESTED** [review]

File: components/navigation.css (hunk 2 above)
Comment (Hamza-777) [reply components/navigation.css]: Oh thank you for pointing it out it was there by mistake.

File: components/card.css (hunk 1 above)
Comment (Hamza-777) [reply components/card.css:173]: sure thing.

**Review by vrinda-mahajan — APPROVED** [review]
LGTM👍

**Comment by Hamza-777** [comment]
> LGTM👍

Thank you. I will go ahead and merge now.

**Pull request merged**